spec/metal-api.json

```diff
@@ -1,5494 +1,5729 @@
-                },
-                "hardware": {
-                    "$ref": "#/definitions/v1.MachineHardware",
-                    "description": "the hardware of this machine"
-                },
-                "id": {
-                    "description": "the unique ID of this entity",
-                    "type": "string",
-                    "uniqueItems": true
-                },
-                "ledstate": {
-                    "$ref": "#/definitions/v1.ChassisIdentifyLEDState",
-                    "description": "the state of this chassis identify LED"
-                },
-                "liveliness": {
-                    "description": "the liveliness of this machine",
-                    "type": "string"
-                },
-                "name": {
-                    "description": "a readable name for this entity",
-                    "type": "string"
-                },
-                "partition": {
-                    "$ref": "#/definitions/v1.PartitionResponse",
-                    "description": "the partition assigned to this machine",
-                    "readOnly": true
-                },
-                "rackid": {
-                    "description": "the rack assigned to this machine",
-                    "readOnly": true,
-                    "type": "string"
-                },
-                "size": {
-                    "$ref": "#/definitions/v1.SizeResponse",
-                    "description": "the size of this machine",
-                    "readOnly": true
-                },
-                "state": {
-                    "$ref": "#/definitions/v1.MachineState",
-                    "description": "the state of this machine"
-                },
-                "tags": {
-                    "description": "tags for this machine",
-                    "items": {
-                        "type": "string"
-                    },
-                    "type": "array"
-                }
-            },
-            "required": [
-                "allocation",
-                "bios",
-                "changed",
-                "created",
-                "events",
-                "hardware",
-                "id",
-                "ledstate",
-                "liveliness",
-                "partition",
-                "rackid",
-                "size",
-                "state",
-                "tags"
-            ]
-        },
-        "v1.MachineState": {
-            "properties": {
-                "description": {
-                    "description": "a description why this machine is in the given state",
-                    "type": "string"
-                },
-                "value": {
-                    "description": "the state of this machine. empty means available for all",
-                    "type": "string"
-                }
-            },
-            "required": [
-                "description",
-                "value"
-            
```

Comment:
Huh? Why is this file completely different now? Where is the difference between the `jq` magic and the Python magic?